impress-org/givewp
# Fix: include honorific with new donors during donation form request

Resolves: [GIVE-32](https://stellarwp.atlassian.net/browse/GIVE-32)

## Description



This includes the donor prefix/honorific for _new_ donors during the donation form request.

Note: this does not include updating the honorific for existing donors.  The way that worked in the past did not align with our philosophical separation of donor details and donation billing details.  In a subsequent PR we will explore adding a path for a new approach, wherein the honorific in the request for an existing donor, will be saved to the donation record either via the first name meta or a new meta.

## Affects



- v2 and v3 donation form requests.

## Visuals



N/A

## Testing Instructions



- Create v2 and v3 forms with honorific / donor title prefix enabled
- Create donations in incognito using unique emails with honorific selected to trigger new donors
- Check donor record to make sure honorific is stored

## Pre-review Checklist






[GIVE-32]: https://stellarwp.atlassian.net/browse/GIVE-32?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
